vibe_code_apps_19/app/data_contract.py

```
@@ -505,6 +548,36 @@ def audit_package_dir(
+                        severity="warn" if plaus or reasons.get("SENTINEL") else "info",
+                        message=(
+                            f"{eid}: {fq.invalid_sample_count} invalid sample(s) "
+                            f"(coverage {fq.valid_coverage:.1%}; reasons={reasons})"
+                        ),
+                        equipment_id=eid,
+                        count=int(fq.invalid_sample_count),
+                        samples=list(reasons)[:8],
+                    )
+                )
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)